server/gitrest/packages/gitrest-base/package.json

```diff
@@ -27,7 +27,7 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "@fluidframework/common-utils": "^0.32.1",
+    "@fluidframework/common-utils": "^0.33.1000-74526",
```

Comment:
This is OK, but I think it's better to use a version range like this:

`^0.33.1000-0`

That will make it easier to pull in new pre-release versions -- only the lockfiles would need to be updated.

But don't change it in this case, especially since CI is passing as-is and a change would require a full CI run.

Reply:
I disagree... I think this way makes it easier to pull in subsequent pre-release versions, because you _don't_ have to mess with the lock files :)  And I like that it's explicit in the package.json which pre-release was pulled in - makes it somewhat possible to trace back and see what code change in the dependency was targeted.